Dockerfile.fpm

```diff
@@ -25,8 +26,4 @@ COPY --from=composer --chown=elife:elife /app/vendor/ vendor/
 COPY --chown=elife:elife src/ src/
 COPY --chown=elife:elife test/ test/
 
-RUN bin/console assets:install
```

Comment:
We will need this, but we currently only use it in `dev`.

Reply:
Looks like running it in `prod` actually makes no difference, they still get installed anyway (like https://elifesciences.org/bundles/csaguzzle/js/guzzle.min.js).